mwdb/model/object.py

```diff
@@ -497,3 +506,9 @@ def get_shares(self):
                       .order_by(ObjectPermission.access_time.desc())
         ).all()
         return shares
+
+    def is_favorite_object(self, user):
+        if user in self.followers:
+            return True
+        else:
+            return False
```

Comment:
I would change that to property, so we can expose it via `GET /<type>/<hash>`
```suggestion
    @property
    def favorite(self):
        return g.auth_user in self.followers
```

Reply:
Then just add it to `ObjectItemResponseSchema` and check if it works.